simulator/task_manager_test.go

```diff
@@ -14,34 +14,167 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package simulator
+package simulator_test
 
 import (
+	"context"
 	"testing"
 
-	"github.com/vmware/govmomi/simulator/esx"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/property"
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/task"
+	"github.com/vmware/govmomi/vim25"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
 )
 
 func TestTaskManagerRecent(t *testing.T) {
```

Comment:
Convert to an `Example` test pattern?

Reply:
There is a simpler example here, which now runs against vcsim with this PR: https://github.com/vmware/govmomi/blob/main/examples/tasks/main.go